qt/Torrent.h

```diff
@@ -320,6 +320,21 @@ class Torrent : public QObject
         return s ? static_cast<double>(s - l) / static_cast<double>(s) : 0.0;
     }
 
+    [[nodiscard]] constexpr double seedRatioPercentDone() const noexcept
+    {
+        auto const r = ratio();
+        auto const g = getSeedRatioLimit();
+        return static_cast<double>(r) / static_cast<double>(g.value_or(r));
+    }
+
+    [[nodiscard]] constexpr double activityPercentDone() const noexcept
+    {
+        if (isDone())
+            return seedRatioPercentDone();
+        else
+            return percentDone();
```

Comment:
_(nit, style)_

In general, the codebase avoids return-after-else and uses ternaries when possible

```suggestion
        return isDone() ? seedRatioPercentDone() : percentDone();
```